ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -260,8 +260,8 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task
  completionHandler:(ChallengeCompletionHandler)completionHandler
 {
     NSString *authMethod = [challenge.protectionSpace.authenticationMethod lowercaseString];
-    AD_LOG_VERBOSE_F(@"session:task:didReceiveChallenge:completionHandler", _context.correlationId,
-                     @"%@. Previous challenge failure count: %ld", authMethod, (long)challenge.previousFailureCount);
+    
+    AD_LOG_VERBOSE(_context.correlationId, nil, NO, @"%@ - %@. Previous challenge failure count: %ld", @"session:task:didReceiveChallenge:completionHandler", authMethod, (long)challenge.previousFailureCount);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 208 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)